tests/installation/select_patterns_and_packages.pm

```diff
@@ -60,6 +60,11 @@ sub gotopatterns {
         send_key 'alt-s';
     }
     else {
+        # See poo#12322. Prevent tab presses before overview is fully loaded
+        if (!check_screen('installation-settings-overview-loaded')) {
```

Comment:
my point here that you will get into same place not only when it is slow machine but also if by any reason you was stack in selecting package flow (sub package_action ), and in this case your message will mislead instead of helping. 
In other words you have not done enough to make sure that you exactly in state about which you giving message